sql/2020/09_Performance/lh6_vs_lh5_performance_score_01.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+
+# Calculates minimum, maximum and average delta between LH5 and LH6 performance score for mobile
+SELECT MIN(perf_score_delta) AS min_delta, MAX(perf_score_delta) AS max_delta, AVG(abs_perf_score_delta) AS avg_delta
```

Comment:
Format so that each field is on its own line.

Also, let's use percentiles here. See https://github.com/HTTPArchive/almanac.httparchive.org/blob/dbb6d5836fee99472fc879132e74f86142a585f8/sql/2019/07_Performance/07_10.sql for an example query. IN this case I think we should do `APPROX_QUANTILES(perf_score_delta, 1000)[OFFSET(percentile * 10)]`